pandas/core/generic.py

```diff
@@ -5255,6 +5255,9 @@ def abs(self):
 
     @Appender(_shared_docs['describe'] % _shared_doc_kwargs)
     def describe(self, percentiles=None, include=None, exclude=None):
+        # explicit conversion of `percentiles` to list
+        percentiles = list(percentiles)
```

Comment:
Right - I think I could just move it under `if percentiles is not None:` instead